packages/@aws-cdk/aws-apigateway/lib/method.ts

```diff
@@ -65,15 +65,45 @@ export interface MethodOptions {
   readonly requestParameters?: { [param: string]: boolean };
 
   /**
-   * The resources that are used for the response's content type. Specify request
-   * models as key-value pairs (string-to-string mapping), with a content type
-   * as the key and a Model resource name as the value
+   * The models which describe data structure of request payload. This can work
+   * together with `requestValidator` or `requestValidatorOptions` to validate
+   * the API request payload before it reaches integrations like Lambda Proxy
+   * Integration. Specify `requestModels` as key-value pairs, with a content type
```

Comment:
```suggestion
   * The models which describe data structure of request payload. When
   * combined with `requestValidator` or `requestValidatorOptions`, the service
   * will validate the API request payload before it reaches the API's Integration.
   * Specify `requestModels` as key-value pairs, with a content type
```

Reply:
Don't remove `Lambda Proxy Integration`. Users might think that it'll not work with Lambda Proxy Integration because it's a proxy. In reality, it works with Lambda Proxy Integration and it saves lots of manual validation checks in lambda code.

Reply:
How about something like this then?

```suggestion
   * The models which describe data structure of request payload. When
   * combined with `requestValidator` or `requestValidatorOptions`, the service
   * will validate the API request payload before it reaches the API's Integration (including proxies).
   * Specify `requestModels` as key-value pairs, with a content type
```